pom.xml

```diff
@@ -109,6 +110,13 @@
           <version>${grpc.version}</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-tcnative-boringssl-static</artifactId>
+          <version>${netty-tcnative.version}</version>
+          <classifier>${os.detected.classifier}</classifier>
+          <scope>test</scope>
```

Comment:
Sorry I am not an export on maven dependency. More question, If I scope the `netty-tcnative-boringssl-static` to test. then the compiled scope of jetcd will not have `io.netty.handler.ssl.SslContext` defined anywhere. this means that the user must import another jar that has `io.netty.handler.ssl.SslContext`?